pkgs/development/python-modules/jenkinsapi/default.nix

```
@@ -19,14 +22,19 @@ buildPythonPackage rec {
   };
 
   propagatedBuildInputs = [ pytz requests ];
-  buildInputs = [ coverage mock nose unittest2 ];
+  checkInputs = [ mock pycodestyle pylint pytest pytest-mock requests-kerberos toml ];
+  checkPhase = ''
+  pylint jenkinsapi
+  pycodestyle
+  # don't run tests that try to spin up jenkins, and a few more that are mysteriously broken
+  py.test jenkinsapi_tests -k "not systests and not test_plugins and not test_view"
+  '';
```

linting is useful upstream, but not as much to verify correctness, you can probably skip it.

Generally comments are put outside of code blocks, as this won't trigger rebuilds when the comments are changed. Also, please indent. Line break is optional :)
```suggestion
  # don't run tests that try to spin up jenkins, and a few more that are mysteriously broken
  checkPhase = ''
    py.test jenkinsapi_tests \
      -k "not systests and not test_plugins and not test_view"
  '';
```

---

ooh nice, I didn't know/think of that